src/systems/systems/src/air_conditioning/acs_controller.rs

```diff
@@ -668,14 +668,16 @@ impl<const ZONES: usize> PackFlowController<ZONES> {
             && !(pneumatic.left_engine_state() == EngineState::Starting)
             && (!(pneumatic.right_engine_state() == EngineState::Starting)
                 || !pneumatic.engine_crossbleed_is_on())
+            && (pneumatic.engine_mode_selector() != EngineModeSelector::Ignition)
             && !engine_fire_push_buttons.is_released(1)
             && !pressurization_overhead.ditching_is_on();
-        // && ! pack 1 overheat
-        // Flow Control Valve 2
-        self.fcv_2_open_allowed = acs_overhead.pack_pushbuttons_state()[1]
+            // && ! pack 1 overheat
+            // Flow Control Valve 2
+            self.fcv_2_open_allowed = acs_overhead.pack_pushbuttons_state()[1]
```

Comment:
```suggestion
        // && ! pack 1 overheat
        // Flow Control Valve 2
        self.fcv_2_open_allowed = acs_overhead.pack_pushbuttons_state()[1]
```
Indentation was correct before?